weekCategories/WeekCategoryComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+
+export default function CategoryButton(props:any) {
+    var [categoryContext,setCategoryContext] = useState(props.value);
+   return(
+    <View style={[styles.screenContainer] }>
+    <Text style={styles.myFontColor}>{categoryContext+"  "} 
```

Comment:
Do we have the two whitespaces here to horizontally align the `categoryContext` with the `x` button?